src/site/content/en/blog/advancing-framework-ecosystem-cds-2019/index.md

```diff
@@ -0,0 +1,208 @@
+  </figcaption>
+</figure>
+
+We are also exploring other features to improve both the user and developer experience of using
+Next.js, such as:  
+
++   Enabling concurrent mode to unlock progressive or partial hydration of components. 
++   A webpack based conformance system that analyzes all source files and generated assets to
+    surface better errors and warnings ([RFC](https://github.com/zeit/next.js/issues/9310)).
+
+<figure class="w-figure w-figure--center">
+  <img src="conformance.png" class="w-screenshot" alt="Example of a conformance build error in Next.js">
+  <figcaption class="w-figcaption">
+    An example of a conformance build error in Next.js (prototype)
+  </figcaption>
+</figure>
+
+## Nuxt.js
+
+[Nuxt.js](https://nuxtjs.org/) is a web framework that combines Vue.js with different libraries to
+provide an opinionated setup. Similar to Next.js, it includes many features out-of-the-box:
+
++   Routing with default code-splitting
++   Compilation and bundling (using [Babel](https://babeljs.io/) and
+    [webpack](https://webpack.js.org/))
++   Server-side rendering
++   Asynchronous data fetching for every page
++   Default data store ([Vuex](https://vuex.vuejs.org/guide/))
+
+Along with working directly on improving the performance of different tools, we've expanded the
+framework fund to provide monetary support to more open-source frameworks and libraries. With our
+[recent support](https://github.com/nuxt/nuxt.js/issues/6467#issuecomment-538192059) to Nuxt.js, a
+few features are slated to land in the near future including smarter server-rendering and image
+optimizations.
+
+{% Aside %}
+The [framework fund](https://opencollective.com/chrome) accelerates the efforts of different
+frameworks and libraries with the goal to improve their performance. If you are working on an
+open-source tool and need support, [please apply](http://bit.ly/chrome-framework-fund)!
+{% endAside %}
+
+## Babel
+
+We've also made progress on improving the performance of an important underlying tool in almost all
+of the mentioned frameworks--[Babel](https://babeljs.io/).
+
+Babel compiles code that contains newer syntax into code that different browsers can understand.
+It's become common to use [@babel/preset-env](https://babeljs.io/docs/en/babel-preset-env) to target
+modern browsers where different browser targets can be specified to provide enough polyfilling
+required for all the chosen environments. One way to specify the targets is to use `<script
+type="module">` to target all browsers that [support ES
+Modules](https://babeljs.io/docs/en/babel-preset-env#targetsesmodules).
+
+{% Aside %}
+To learn more about how to leverage ES modules to minimize polyfill code shipped to browsers
+that do not need them check out
+["Serve modern code to modern browsers for faster page loads"](../serve-modern-code-to-modern-browsers).
```

Comment:
```suggestion
[Serve modern code to modern browsers for faster page loads](/serve-modern-code-to-modern-browsers).
```

No need for quotes around titles because the link styling sets them apart

Also changed the path to be absolute